base/audit_events.go

```diff
@@ -22,20 +34,42 @@ const (
 //   - a kv-auditd-compatible descriptor with TestGenerateAuditdModuleDescriptor
 //   - CSV output for each event to be used to document
 var AuditEvents = events{
-	AuditIDPlaceholder: {
-		Name:        "Placeholder audit event",
-		Description: "This is a placeholder.",
-		MandatoryFields: map[string]any{
-			"context": map[string]any{
-				"provider": "example provider",
-				"username": "alice",
-			},
+	AuditIDAuditEnabled: {}, // TODO: unreferenced event - somehow find this mistake via test or lint!
```

Comment:
Do you want to file a ticket for this?

Reply:
https://issues.couchbase.com/browse/CBG-4037